input/postgres/statements.go

```diff
@@ -230,6 +234,18 @@ func GetStatements(ctx context.Context, server *state.Server, logger *util.Logge
 	statementTextsByFp := make(state.PostgresStatementTextMap)
 	statementStats := make(state.PostgresStatementStatsMap)
 
+	queryKeys := make([]state.PostgresStatementKey, 0)
+	queryStats := make([]state.PostgresStatementStats, 0)
+	queryLength := make([]int, 0)
+	if showtext {
+		tmpFile, err = ioutil.TempFile("", "")
```

Comment:
This function is deprecated in Go now; it just [calls another function](https://cs.opensource.google/go/go/+/refs/tags/go1.23.2:src/io/ioutil/tempfile.go;l=23-26).
```suggestion
		tmpFile, err = os.CreateTemp("", "")
```
I'll clean up other usage, but we should avoid introducing more.